### PR TITLE
Site improvements

### PR DESCRIPTION
**Pull request by sexybiggetje**
Fix a small grammar on FAQ.
Add some breathing space by increasing margins.
Add a bit of font-size to body texts to increase readability.
Increase lineheight for readability.
Make the menu a fixed size so it flows better.
Also icons arent all of the same width, so realigned the menu properly to match that.

### CODE REVIEW

**Comment by andy5995** [comment]
@sexybiggetje I think you were right that some old commits got mixed in. The first 2, and the third shows a commit for "sync to master"...

When you sync, it should never show as a commit otherwise it makes things uneven.

See https://help.github.com/en/articles/syncing-a-fork for a suggested method

File: docs/faq.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: FAQ
 layout: default
 ---
 
-**What your release cycle?**
+**What is your release cycle?**
```
Comment (andy5995) [inline docs/faq.md]: If a compiler can detect problems, why can't we make something that can scan a document for grammar problems? xD (project idea).

Comment (sexybiggetje) [reply docs/faq.md]: see @ifmeorg/ifme-languagetools :)

**Comment by sexybiggetje** [comment]
Ok when I'm on an authenticated machine I can rebase from HEAD~4 to skip the first 3 probably.

**Comment by sexybiggetje** [comment]
@andy5995 how does this latest change look? If this doesnt look right I`ll just refork and copy my changes into this before I completely mess things up ;)

**Comment by andy5995** [comment]
> @andy5995 how does this latest change look? If this doesnt look right I`ll just refork and copy my changes into this before I completely mess things up ;)

It's showing changes to po/nl_NL.po and they are incorrect. Those variables aren't included in the po file anymore.

I think the easiest way to fix is copy the nl_NL.po file from the repo and overwrite the one you have on this branch. Then push, and that file will then disappear from the "changed files" in the PR.

File: po/nl_NL.po [hunk 2]
```diff
@@ -19,63 +19,39 @@ msgstr ""
...
-#, fuzzy
-#| msgid ""
-#| "\n"
-#| "# A folder can use the $UID variable.\n"
-#| "# WASTE=/mnt/fs/Trash-$UID\n"
+#: src/config_rmw.c:73
 msgid ""
 "\n"
 "# A folder can use the $UID variable.\n"
+"# WASTE=/mnt/fs/Trash-$UID\n"
```
Comment (andy5995) [inline po/nl_NL.po]: This is one of the variables I meant in my comment a minute ago. Not included in the translation anymore

File: po/nl_NL.po [hunk 3]
```diff
@@ -19,63 +19,39 @@ msgstr ""
...
+#: src/config_rmw.c:89
+#, c-format
 msgid ""
 "\n"
 "# How many days should files be allowed to stay in the waste folders before\n"
 "# they are permanently deleted\n"
 "#\n"
 "# use '0' to disable purging\n"
 "#\n"
+"purge_after = %d\n"
```
Comment (andy5995) [inline po/nl_NL.po]: And this....

File: po/nl_NL.po [hunk 4]
```diff
@@ -19,63 +19,39 @@ msgstr ""
...
 msgstr ""
 "\n"
 "# Hoeveel dagen zouden bestanden mogen blijven staan in de afval map\n"
 "# voordat ze permanent worden verwijderd\n"
 "#\n"
 "# gebruik '0' om purging uit te schakelen\n"
 "#\n"
+"purge_after = %d\n"
```
Comment (andy5995) [inline po/nl_NL.po]: And this...

File: po/nl_NL.po [hunk 5]
```diff
@@ -91,6 +67,7 @@ msgid ""
 "# Note to OSX and Windows users: sending files to 'Desktop' trash\n"
 "# doesn't work yet\n"
 "#\n"
+"# WASTE=$HOME/.local/share/Trash\n"
```
Comment (andy5995) [inline po/nl_NL.po]: And this...

File: po/nl_NL.po [hunk 6]
```diff
@@ -103,26 +80,20 @@ msgstr ""
...
+#, c-format
 msgid ""
 "\n"
 "# Removable media: If a folder has ',removable' appended to it, rmw\n"
 "# will not try to create it; it must be initially created manually. If\n"
 "# the folder exists when rmw is run, it will be used; if not, it will be\n"
 "# skipped Once you create \"example_waste\", rmw will automatically create\n"
 "# example_waste/info and example_waste/files\n"
+"#\n"
+"# WASTE=/mnt/sda10000/example_waste, removable"
```
Comment (andy5995) [inline po/nl_NL.po]: And this

File: po/nl_NL.po [hunk 7]
```diff
@@ -134,27 +105,27 @@ msgstr ""
...
-#| "#\n"
+#. TRANSLATORS:  Do not translate the last line in this section
+#: src/config_rmw.c:99
+#, c-format
 msgid ""
 "\n"
 "# purge is allowed to run without the '-f' option. If you'd rather\n"
 "# require the use of '-f', you may uncomment the line below.\n"
 "#\n"
+"# force_required\n"
```
Comment (andy5995) [inline po/nl_NL.po]: And this.. and the related ones xD

**Comment by sexybiggetje** [comment]
copied over nl_NL.po from master

**Pull request merged**